### PR TITLE
design(1500): FIx sd-zip-artifacts design

### PR DESCRIPTION
**Pull request by kkisic**
## Context
Fix SD_ZIP_ARTIFACTS design.

## Objective
- Add the process to delete zip artifact to unzip worker
- Add DELETE endpoint to store
- Make the picture in the overview detail

## References
https://github.com/screwdriver-cd/screwdriver/issues/1500

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by sugarnaoming — DISMISSED** [review]

File: design/sd-zip-artifacts.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Status(Error):
 
 |Method|URL|Description|
 |:--|:--|:--|
-|POST|/queue/message|enqueue message to unzip file|
+|POST|/queue/message?type=unzip|enqueue message to unzip file|
```
Comment (yuichi10) [inline design/sd-zip-artifacts.md:94]: [imo]
Maybe we do not need to add parameters here. Parameters are explained in POST DATA section.

Comment (kkisic) [reply design/sd-zip-artifacts.md]: Thx.
I deleted it.

File: design/sd-zip-artifacts.md [hunk 2]
```diff
@@ -134,12 +134,21 @@ This is a new component.
...
+1. Upload unzipped artifact files
+1. Delete SD_ARTIFACT.zip
 
-1. Upload and Download artifact files
+#### API
 
-SD Store do not need to add new function, but need to add new Authentication.
+|Method|URL|Description|
+|:--|:--|:--|
+|DELETE|/builds/{id}/ARTIFACTS/SD_ARTIFACT.zip|delete zipped artifact files|
```
Comment (yuichi10) [inline design/sd-zip-artifacts.md:149]: I feel a little bit strange the path of `SD_ARTIFACT.zip`.
Is it better to path filename with body or parameter?

Comment (kkisic) [reply design/sd-zip-artifacts.md:149]: I'm using the same implementation as `DELETE /caches` and `DELETE /commands`.
- https://github.com/screwdriver-cd/store/blob/176fc403741f462d95a2f0bfe1dae26fd724f5a1/plugins/caches.js#L303-L434
- https://github.com/screwdriver-cd/store/blob/176fc403741f462d95a2f0bfe1dae26fd724f5a1/plugins/commands.js#L165-L212

Comment (yuichi10) [reply design/sd-zip-artifacts.md:149]: When I implement this API literally the path will be `path: '/builds/{id}/SD_ARTIFACT.zip` not like `path: '/builds/{id}/{filename}`. 

There is a possibility that the zip file name will change,  at that time the path of `SD_ARTIFACT.zip` is strange.

Comment (kkisic) [reply design/sd-zip-artifacts.md]: I fixed the path.

**Review by yuichi10 — APPROVED** [review]
LGTM

**Review by tkyi — APPROVED** [review]

**Pull request merged**